### ISSUE TITLE
integrate rcsd functionality into dwi2fod

### ISSUE THREAD

**Issue by jdtournier** [MRtrix3/mrtrix3#190]
I think this is probably the cleanest way to introduce the benefits of rcsdeconv into the core of MRtrix. The current version of rcsdeconv is a little bit hard to follow in terms of the code, and most of the features required are relatively simple to add to the current version of dwi2fod - exception being the neighbourhood regularisation, but I'm thinking of splitting that off into a separate app, which would produce the neighbourhood-smoothed of the output for inclusion as a prior in a subsequent call to dwi2fod...

**Comment by jdtournier** [comment]
OK, looks like that was my goof... All fixed now.

**Comment by Lestropie** [comment]
... Maaaybe the wrong issue? :-P

**Comment by jdtournier** [comment]
Ah crap.

**Comment by Lestropie** [comment]
Removed release milestone following discussion at ISMRM2016. Settling with the outcomes of #604 for now, can revisit this after release.

**Comment by jdtournier** [comment]
Don't think is ever going to happen. Closing...
